assets_management/models/account_account.py

```diff
@@ -2,28 +2,35 @@
 # Copyright 2019 Openforce Srls Unipersonale (www.openforce.it)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import _, api, models
+from odoo import _, models
 from odoo.exceptions import UserError
 
 
 class AccountAccount(models.Model):
-    _inherit = 'account.account'
+    _inherit = "account.account"
 
-    @api.multi
```

Comment:
Le modifiche relative al porting andrebbero separate in un loro commit, diverso da quello 'black, isort, prettier'. Vd. per es. gli ultimi due commit in https://github.com/OCA/l10n-italy/pull/2848/commits